apstools/utils/device_info.py

```diff
@@ -7,10 +7,10 @@
    ~listdevice
 """
 
-
 import datetime
-import logging
 from collections import defaultdict
+import logging
+from typing import Any, Dict, List, Optional, Union
```

Comment:
`isort --sl` (and other config) will change this to single-line imports.  That's the project style.